augly/image/functional.py

```diff
@@ -1036,9 +1036,10 @@ def overlay_emoji(
     output_path: Optional[str] = None,
     emoji_path: str = utils.EMOJI_PATH,
     opacity: float = 1.0,
-    emoji_size: float = 0.15,
-    x_pos: float = 0.4,
-    y_pos: float = 0.8,
+    emoji_size: Union[float, Tuple[float, float]] = 0.15,
```

Comment:
overall question: so currently, `overlay_emoji` is *not* a random function, however `RandomEmojiOverlay` is. `RandomOverlayEmoji` currently randomly chooses an emoji to overlay, however the code here lets `overlay_emoji` do the randomness.  thus, the value setting is spread across two functions -- we want to avoid this.

personally, my suggestion would be to revert these changes in `functional.py` and allow the additional random parameter/seed setting to occur in `transforms.py` :) thoughts? 

cc: @zpapakipos

Reply:
Yes, I think I agree that would be the best. I thought about this before too, and I didn't suggest that change just because I feel like ideally at some point we should add random sampling abilities for all the transform classes -- e.g. be able to pass in ranges for each param, or have a method `sample_parameters()` which can be called after `__init__()` before `__call__()` with a seed.

For now, indeed I think it's better in this PR to take in the param ranges in `RandomOverlayEmoji` but only take in the params as floats in `overlay_emoji()`. So all the changes in `functional.py` should be reverted, and the seed setting & param sampling should be moved to right after line 2392 in `transforms.py`.